projects/aca-folder-rules/src/lib/rule-details/actions/rule-action.ui-component.ts

```diff
@@ -182,8 +195,19 @@ export class RuleActionUiComponent implements ControlValueAccessor, OnInit, OnDe
             ...cardViewPropertiesModel,
             value: this.parameters[paramDef.name] ?? false
           });
+        case 'd:noderef':
+          if (!constraintsForDropdownBox && !this.readOnly) {
+            return new CardViewTextItemModel({
+              ...cardViewPropertiesModel,
+              icon: 'folder',
+              default: this.translate.instant('ACA_FOLDER_RULES.RULE_DETAILS.PLACEHOLDER.CHOOSE_FOLDER'),
+              clickable: true,
+              clickCallBack: this.openSelectorDialog.bind(this),
+              value: this.parameters[paramDef.name]
+            });
+          }
+        //  falls through
```

Comment:
Is that comment needed?

Reply:
Yes, it is needed for linter. Otherwise it gives "_Expected a 'break' statement before 'default_'" error.

Reply:
And you don't need break right? Because it says about break probably because if that condition in if: "if (!constraintsForDropdownBox && !this.readOnly)" gives false then it will go to next case which is default if break is missing.

Reply:
Right